static/js/common.js

```diff
@@ -58,8 +58,25 @@ function password_quality(password, bar, password_field) {
     return acceptable;
 }
 
+function password_warning(password, password_field) {
+    if (typeof zxcvbn === 'undefined') {
+        return undefined;
+    }
+
+    var min_password_length = 6;
```

Comment:
@adnrs96 I think the main reason why we are specifying it again is because data-attributes are not supported in Internet Explorer 10 and below versions.

https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes